docs/content.zh/docs/dev/table/hiveCompatibility/hiveDialect/set.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "SET Statements"
+weight: 8
+type: docs
+aliases:
+- /dev/table/hiveCompatibility/hiveDialect/create.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# SET Statements
+
+## Description
+
+The `SET` statement sets a property which provide a ways to set variables for a session and
+configuration property including system variable and Hive configuration.
+But environment variable can't be set via `SET` statement. The behavior of `SET` with Hive dialect is compatible to Hive's.
+
+## Syntax
+
+```sql
+SET key=value;
```

Comment:
The examples are much different from the syntax...

Reply:
remove the `syntax` and just give a example